SINGE_params.cfg

```diff
@@ -0,0 +1,19 @@
+1 ID   541
```

Comment:
Should we make this file tab-separated as well?  I see multiple spaces instead of tabs.

We will also document this file in the readme.